images/compose/compose.yml

```diff
@@ -56,14 +95,16 @@ services:
         condition: service_healthy
       postgres:
         condition: service_healthy
+      migration_service:
+        condition: service_completed_successfully
+      signing_key_service:
+        condition: service_completed_successfully
```

Comment:
I fear, this won't work with podman-compose, unless we keep the containers running there.

Reply:
Yeah, it won't work with podman-compose for now. The idea is to enable docker-compose to use it and emulate the same behavior using the `wait_for_`* scripts.